app/assets/javascripts/discourse/lib/user-search.js.es6

```diff
@@ -44,7 +44,7 @@ function organizeResults(r, options) {
   if (r === CANCELLED_STATUS) { return r; }
 
   var exclude = options.exclude || [],
-      limit = options.limit || 5,
+      limit = options.limit || 6,
```

Comment:
this is just a bit off, the number of results are limited to 5... not 6. I think the change just creates confusion.